spark/src/test/java/io/ddf/spark/content/ViewHandlerTest.java

```diff
@@ -82,4 +82,85 @@ public void testSubsettingWithGrepIgnoreCase() throws DDFException {
     DDF ddf2 = ddf.VIEWS.subset(columns, grep);
     Assert.assertEquals(2, ddf2.getNumRows());
   }
+
+  @Test
+  public void testSampleWithSize() throws DDFException {
+    createTableAirline();
+
+    DDF ddf = manager.sql2ddf("select * from airline", "SparkSQL");
+    int sampleSize = 2;
```

Comment:
Can we use sampleSize bigger than ddf.nrow, and make sure that without replacement, the return sample has nrow == ddf.nrow, and with replacement, the sample has nrow == sampleSize